pyaedt/icepak.py

```diff
@@ -1092,18 +1092,32 @@ def create_parametric_fin_heat_sink(
         center : list, optional
            List of ``[x, y, z]`` coordinates for the center of
            the heatsink. The default is ``[0, 0, 0]``.
-        plane_enum : optional
+        plane_enum : str, int, optional
+            Plane for orienting the heat sink.
+            :class:`pyaedt.constants.PLANE` Enumerator can be used as input.
             The default is ``0``.
-        rotation : optional
+        rotation : int, float, optional
             The default is ``0``.
-        tolerance : optional
+        tolerance : int, float, optional
             Tolerance value. The default is ``0.001``.
 
         Returns
         -------
         bool
             ``True`` when successful, ``False`` when failed.
 
+        Examples
+        --------
+
```

Comment:
I don't think a blank line is needed here. I'm not sure if it will cause a problem.

Reply:
No it's not. good catch.